lib/mandrill_dm/message.rb

```diff
@@ -276,6 +276,15 @@ def inline_attachments?
       mail.attachments.any?(&:inline?)
     end
 
+    def html_content?
+      return true if mail.content_type.nil?
+      mail.content_type =~ %r{text/html} ? true : false
```

Comment:
Sorry I've haven't looked at this sooner.  Been super busy with work :/  @sysqa I think if you drop the guard-clause `return true if mail.content_type.nil?`, this PR is good to go then I'll release an updated version.